ukoly/RadkaMekutova/ukol-4/ukol.js

```diff
@@ -0,0 +1,10 @@
+/*
+Úkol blog
+
+	• Menu - pole s kategoriema, funkce cretaeMenu, která projde pole a pro kazdou polozku vytvori odkaz a vlozi ho do menu
```

Comment:
Další přístup by byl vytvořit kategorie jako samostatný model stejně jako jsou články.

Např. struktura bude `id`, `name`... takže třeba `id:1`, `name:World`

Pak se ale musí vyřešit párování článek x kategorie. Zde je několik způsobů.
- atribut kategorie bude ID

```json
{
  "id": 1,
  "title": "Můj první článek",
  "category": 2
}
```

- celá struktura kategorie

```json
{
  "id": 1,
  "title": "Můj první článe",
  "category": {
    "id": 2,
    "name": "World"
  }
}
```

- klíčové slovo, které nám bude pomáhat vyhledat články v kategorii

```json
{
  "id": 1,
  "title": "Můj první článek",
  "category": "world"
}
```

Uložení kategorie bych směřoval k tomu, jak budeš podle něj vyhledávat.